app/models/application_record.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  scope :registers_available, -> {
```

Comment:
I don't think we need this in `application_record` can it just be in the registers model if at all?